provisioning/transport/mqtt/src/mqtt.ts

```diff
@@ -269,6 +277,7 @@ export class Mqtt extends EventEmitter implements X509ProvisioningTransport {
       clientId: request.registrationId,
       clean: true,
       x509: this._auth,
+      sharedAccessSignature: this._sasToken,
```

Comment:
>sharedAccessSignature [](start = 6, length = 21)

naming inconsistency catching up to us.... is gonna confuse any potential future contributor.

Reply:
changed things to SharedAccessSignature with acronyms of SAS/sas

---
In reply to: [231364167](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231364167) [](ancestors = 231364167)